Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1364,9 +1366,17 @@ GPUProcessConnection& WebProcess::ensureGPUProcessConnection()
 
     // If we've lost our connection to the GPU process (e.g. it crashed) try to re-establish it.
     if (!m_gpuProcessConnection) {
-        m_gpuProcessConnection = GPUProcessConnection::create(Ref { *parentProcessConnection() });
-        if (!m_gpuProcessConnection)
+        auto connectionIdentifiers = IPC::Connection::createConnectionIdentifierPair();
+        if (!connectionIdentifiers)
             CRASH();
+        RefPtr connection = protectedParentProcessConnection();
```

Comment:
This doesn't need to call `protectedParentProcessConnection()` since you're assigning to a RefPtr. Should call parentProcessConnection().